src/pages/ProductDetail/ProductHeader/ProductHeader.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react';
+import { FiPlusCircle } from 'react-icons/fi';
+import Logo from '../../../assets/ProductDetail/aesop-logo.png';
+import ProductWishList from './ProductWishList/ProductWishList';
+import ProductRelated from './ProductRelated/ProductRelated';
+import ModalDetailView from '../ModalDetailView/ModalDetailView';
+
+import './ProductHeader.scss';
+
+function ProductHeader({ isOpenModal, modalView, slide, isCloseModal }) {
+  const [productDetailList, setProductDetailList] = useState([]);
+  const [wishList, setWishList] = useState(false);
+
+  function handleWIshList() {
+    wishList === false ? setWishList(true) : setWishList(false);
+  }
```

Comment:
```suggestion
  function handleWIshList() {
    setWishList(wishList => !wishList)
  }
```
위와 동일하다고 생각되는데 한 번 고민해보세요!

Reply:
넵!